### ISSUE TITLE
quadpy.c1.gauss_patterson(5) problem

### ISSUE THREAD

**Issue by horvl** [nschloe/quadpy#480]
The example script for the gauss_patterson scheme does not seem to work with the latest version of quadpy. See below.
quadpy version: 0.17.20

```
Python 3.11.4 | packaged by conda-forge | (main, Jun 10 2023, 18:08:17) [GCC 12.2.0]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.14.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import numpy as np
   ...: import quadpy
   ...: 
   ...: scheme = quadpy.c1.gauss_patterson(5)
   ...: scheme.show()
   ...: val = scheme.integrate(lambda x: np.exp(x), [0.0, 1.0])
   ...: 
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
Cell In[1], line 4
      1 import numpy as np
      2 import quadpy
----> 4 scheme = quadpy.c1.gauss_patterson(5)
      5 scheme.show()
      6 val = scheme.integrate(lambda x: np.exp(x), [0.0, 1.0])

File <string>:4100, in _e5qsi(index)

File <string>:4131, in _ap9WK(pts)

File <string>:4131, in <listcomp>(.0)

NameError: name 'i' is not defined

In [2]: 
```

**Comment by nschloe** [comment]
We've reorganized the code recently. Will look at this today.

**Comment by nschloe** [comment]
Fixed in 0.17.21. Thanks for the report!
